arelle/ValidateFilingText.py

```diff
@@ -924,6 +928,23 @@ def warning(self, err):
              error=err.getMessage(), line=err.getLineNumber(), column=err.getColumnNumber())
 '''
 
+@dataclass
+class ImageDataURLParts:
+    mimeSubtype: str | None
+    isBase64: bool
+    data: str
+    def base64GraphicHeaderTypeMatchesMimeSubtype(self) -> bool:
+        headerType = validateGraphicHeaderType(decodeBase64DataImage(self.data))
```

Comment:
Pre-existing, the `split('#')` in `decodeBase64DataImage` seems misplaced since the `#` is the URI fragment, so it seems like that should be separated from the "data" as part of the regex rather than as part of base64 decoding.